### PR TITLE
Css two column layout

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2021-12-04 at 5 55 32 PM

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+</head>
+<body>
+  <div class="container ">
+    <div class="row">
+      <div class="column-full">
+        <h1 class="bighead"> Scale, Speed, Agility</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full second-row">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: The class name `second-row` here gives very little insight into its purpose, you should try to make a more descriptive class name here.

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        <p> We've dedicated thousands of man hours to making healthcare instituitions in the Helium
+          newtowrk run 10x more efficiently and deliver an unparalleled quality of care. </p>
+      </div>
+    </div>
+    <div class="row bigrow">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: The class name `bigrow` here gives very little insight into its purpose, you should try to make a more descriptive class name here. Additionally, class names that include multiple words should have the words separated by dashes, which is known as `kebab-case`.

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      <div class="column-half ">
+        <div class="tleft">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: What does `tleft` mean?

File: css-two-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+          <section>
+            <h3 class="head">
+              Intuitive, <br> Smart Software
+            </h3>
+            <p class="words"> We build products that transform how healthcare facilities run across Africa - making
+              them radically more efficient operationally and financially, complete turnaround
+            </p>
+          </section>
+        </div>
+        <div class="column-half bleft">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: `bleft`?

File: css-two-column-layout/index.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            For Any Circumstance
+            </h3>
+            <p class="words"> Our intuitive system is built around the perculiarites of your facility; we support offline
+            deployments, sync to the cloud and run on any device 24/7.
+            </p>
+          </section>
+        </div>
+      </div>
+      <div class="column-half ">
+        <div class="tright">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: `tright`?

File: css-two-column-layout/index.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+          <section>
+            <h3 class="head">
+                Practice Medicine, Not Administration
+            </h3>
+            <p class="words"> Our all-in-one electronic health records (EHR), hospital managment(HMIS) & billing system
+                helps streamline your facility, reduce waste and improve quality of care.
+            </p>
+          </section>
+        </div>
+        <div class="column-half bright">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: `bright`? Is it shiny?

File: css-two-column-layout/index.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+            </h3>
+            <p class="words"> We understand your facility better than you do; we provide visually-appealing real-time analytics,
+              reports and trends at your finger tips.
+            </p>
+          </section>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full last">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: `last`? `last` what?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Daniel! Nice start on this, I have some feedback about class names to get you started:

**Comment by dcatan89** [comment]
changed class names to be more descriptive

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
These class names are definitely longer, but they don't describe their purpose at all. You should make the class name describe either what the content is (like a description of the contents of the element you are influencing such as `smart-software-section`) or a descriptor of what styling the class is actually doing if you are trying to create a reusable utility class.

File: css-two-column-layout/styles.css [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+.column-full {
+  width: 100%;
+}
+
+.column-half {
+  width: 50%;
+}
+
+.smart-software-section {
+  display: flex;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: I see you are using `display: flex` for each of these sections. When you are using layout classes to control your app layout, `display: flex` is usually an indicator that you need another `row`. You should put a `row` class element inside the `column` these classes are being applied to, then have additional `column` classes for the content inside that `row` It's always okay to have additional `row`s inside of a `column` class element 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**